src/commcare_cloud/ansible/roles/rabbitmq/tasks/main.yml

```diff
@@ -40,19 +40,19 @@
   become: yes
   service: name=rabbitmq-server state=started
 
-- include: config-cluster.yml
+- include_tasks: config-cluster.yml
   when: rabbitmq_create_cluster|bool
 
-- include: config.yml
+- include_tasks: config.yml
   when: not rabbitmq_create_cluster or ( ansible_fqdn == hostvars[rabbitmq_cluster_master]['ansible_fqdn'] )
 
 - name: Enable and start RabbitMQ service
   become: yes
   service: name=rabbitmq-server state=started
   tags: after-reboot
 
-- include: join-cluster.yml
+- include_tasks: join-cluster.yml
   when: (rabbitmq_create_cluster | bool) and ( ansible_fqdn != hostvars[rabbitmq_cluster_master]['ansible_fqdn'] )
 
-- include: ha-policy.yml
+- include_tasks: ha-policy.yml
   when: (rabbitmq_create_cluster| bool ) and ( ansible_fqdn == hostvars[rabbitmq_cluster_master]['ansible_fqdn'] )
```

Comment:
the tasks are dynamic because their inclusion depends on the values of rabbitmq_create_cluster, ansible_fqdn, and rabbitmq_cluster_master.